### DISCUSSION TITLE
Better user experience for non reactive sites

### DISCUSSION THREAD

**Discussion by storynory** [tinacms/tinacms#3959] [Ideas]
If you are using a framework like 11ty or Astro that is not reactive, you see a lot of 

Looks like there's
nothing to edit on
this page.

Which is not terribly polished.  My suggestion is that for -non-reactive users, the links under the posts list should go straight through to the edit-in-admin panel. Also there should be a more user-friendly greeting when you first enter the Tina app.      I think this would be in line with what content-editors expect and would avoid a lot of confusion, and maybe widen the appeal of Tinacms, including users migrating from forestry.io - but also all those using static page generators rather than react.    Would it be possible to detect if react is being used, and offer these tweaks?

**Comment by logan-anderson** [comment]
Hello @storynory the preview you are seeing should only show up if the `ui.router` property is being used on a collection.  The router property is only meant for reactive sites. I suggest you remove that property if its present and then all navigation should happen in the admin and it should not show the preview.

**Reply by storynory** [reply]
Thank you that solved it. Sorry for confusion.  It all looks fine now.
